### PR TITLE
Add plugin to add token on context

### PR DESCRIPTION
**Pull request by edsonjab**
Fix issue #71

### CODE REVIEW

**Review by MichaelGHSeg — APPROVED** [review]

**Review by MichaelGHSeg — APPROVED** [review]

**Review by MichaelGHSeg — CHANGES_REQUESTED** [review]
Oops, that caused more test failures

File: packages/core/test/analytics_test.dart [hunk 1]
```diff
@@ -1,7 +1,8 @@
-import 'package:segment_analytics/analytics.dart';
-import 'package:segment_analytics/analytics_platform_interface.dart';
-import 'package:segment_analytics/logger.dart';
-import 'package:segment_analytics/state.dart';
+import 'package:analytics/analytics.dart';
```
Comment (MichaelGHSeg) [inline packages/core/test/analytics_test.dart:1]: This seems to have cause a lot of errors with missing functions etc.

Comment (nd4p90x) [reply packages/core/test/analytics_test.dart:1]: @edsonjab

Comment (edsonjab) [reply packages/core/test/analytics_test.dart]: Hi @MichaelGHSeg I made double check on local must to be works now with the new commit.

**Review by MichaelGHSeg — APPROVED** [review]

**Pull request merged**